src/index.js

```diff
@@ -1,6 +1,27 @@
 const fibonacci = (n) => {
+  if (n < 0) {
+    return console.error(new Error('No puede ser negativo'));
+  }
+
+  let array = [];
+
+  for (let i = 0; i <= n; i++) {
+    array.push(fibo(i));
```

Comment:
Este algoritmo es un poco ineficiente, ya que en cada paso del for de arriba vas calculando toda la serie anterior, que ya tienes calculada. Intentar con recursividad y programación dinámica. Adapta la idea que David hizo en python a javascript de la case https://platzi.com/clases/1835-programacion-estocastica/26430-optimizacion-de-fibonacci/